examples/IRB140/HandInput.m

```diff
@@ -0,0 +1,16 @@
+classdef HandInput < SingletonCoordinateFrame
+  % hand input coordinate frame
+  methods
+    function obj=HandInput(r, ind, name)
+      typecheck(r,'TimeSteppingRigidBodyManipulator');
```

Comment:
i guess you want the time-stepping manipulator to simulate the contact.  but the arm + hand by itself would be cleaner as just a RBM.
Seems like this could be either {'TSRBM','RBM'} -- forgive the shorthand -- or 'IRB140'